onnxruntime/python/tools/quantization/calibrate.py

```diff
@@ -599,19 +599,19 @@ def compute_percentile(self):
 
         print("Number of tensors : {}".format(len(histogram_dict)))
         print("Number of histogram bins : {}".format(self.num_bins))
-        print("Percentile : {}".format(percentile))
+        print("Percentile : ({},{})".format(100.0 - percentile, percentile))
 
         for tensor, histogram in histogram_dict.items():
             hist = histogram[0]
             hist_edges = histogram[1]
             total = hist.sum()
             cdf = np.cumsum(hist/total)
             if self.symmetric:
-                idx_right = np.searchsorted(cdf, percentile/100)
+                idx_right = np.searchsorted(cdf, np.percentile(cdf, percentile))
                 thresholds_dict[tensor] = (-float(hist_edges[idx_right]), float(hist_edges[idx_right]))
             else:
-                idx_right = np.searchsorted(cdf, percentile/200)
-                idx_left = np.searchsorted(cdf, (1.0 - percentile/200))
+                idx_right = np.searchsorted(cdf, np.percentile(cdf, percentile))
```

Comment:
ah, this is because of a math error. It was intent to be:

```
  percent_to_cut_one_side = (100.0 - percentile)/200.0
  idx_right = np.searchsorted(cdf, 1.0 - percent_to_cut_one_side)
  idx_left = np.searchsorted(cdf, percent_to_cut_one_side)
```

Reply:
@mfuntowicz, thanks for the fix. Could you please try if this work? It is simpler. @chilo-ms, we need to add unit test to cover this later.

Reply:
Didn't look at a unit test because I'm not so familiar with the structure of the tests you had in ORT, but will definitively look at it for future PRs, sorry about that.

Reply:
We will add it. Thanks for the fix.